assets/js/index_contact.js

```diff
@@ -0,0 +1,35 @@
+const form = document.querySelector('#contactme');
+
+function validateEmail(email) {
+  if (email.value.trim() === email.value.trim().toLowerCase()) {
+    return true;
+  }
+  return false;
+}
+
+function setMessage(id, message, type) {
+  const divId = document.querySelector(`#${id}`);
+
+  const errorDiv = document.createElement('div');
+  errorDiv.id = `${id}-error-message`;
+  errorDiv.className = `contact-${type}`;
+  errorDiv.textContent = `${message}`;
+
+  form.insertBefore(errorDiv, divId);
+}
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+
+  const errorDiv = document.querySelector('#contact-email-error-message');
+
+  if (errorDiv) {
+    errorDiv.remove();
+  }
+
+  if (!validateEmail(form.elements['contact-email'])) {
+    setMessage('contact-email', 'Ooops!, please write your email in lowercase!', 'error');
+  } else {
+    form.submit();
```

Comment:
Please make sure that the error message is near the submit button as per project requirements.

![image](https://user-images.githubusercontent.com/38088818/195715430-eb41223b-814c-45d0-a552-3216da03b94d.png)

Reply:
It's done:

![image](https://user-images.githubusercontent.com/53025062/195721806-1ca30eb2-d032-4618-9558-8cb82905ec80.png)